### DISCUSSION TITLE
2023 Q2 Community Manager Proposal

### DISCUSSION THREAD

**Discussion by Cindy884** [ringdao-community-guild/discussion#2] [General]
### Work Description
As a DCDAO manager, my primary responsibility is to manage the community, promote DCDAO, and establish strong connections with community members. Here is my Q2 position plan:
1. Community Management:
- Maintain social media platforms, including Twitter, Discord, etc., and regularly post interesting and useful information and activities like web3 latest news or AMA.
- Stay closely connected with community members, answer their questions, address their concerns, and establish strong relationships.
- Organize DCDAO weekly Meetings and update the latest progress with DCDAO’s member
- Review the ambassador’s monthly work in dework and issue the rewards.
2. DCDAO Development:
- Maintain DCDAO to ensure community governance and decision-making.
- Work with developers to implement features such as voting, proposal submission, and budget allocation within the DCDAO.
- Encourage community members to participate in the DCDAO decision-making process and educate them on the benefits and importance of decentralized governance.
3. Community Outreach:
- Reach out to other web3 projects and communities to build partnerships and collaborations.
- Participate in industry events and conferences to represent our platform and network with other DAOs and blockchain projects.
- Develop educational resources and materials to promote decentralized governance and encourage wider adoption of DAOs in the web3 ecosystem.
### Q2 Objectives 
1. Community Management:
- Increase DCDAO's SNS account influence, including Twitter, discord
- Host at least 1 online event with community members per month
- Host DCDAO Weekly meeting to discuss the development of DCDAO
- Input “Monthly Report” about DCDAO contribution
- Input "Monthly Report" as a community manager
2. DAO Development
- To attract more potential members to join DCDAO and build  a sense of identity within the community
- Increase community participation in the DCDAO the decision-making process, implement key DCDAO features such as voting and budget allocation, and establish a system of trust and transparency
3. Community Outreach:
- Develop educational resources and materials, increase engagement with educational content, and increase awareness of the benefits of DCDAO
#### Funding Support
1. Community Manager Monthly Salary: 1700 USD(USDT is preferred)
My metamaskl wallet address: 0xc2B59Ecd40b8B54Bc23Bc395EefA0DEC3CCC6c5F

**Comment by 1022ranji** [comment]
I'm a bit confused. If I understand correctly, the title of this proposal is to write about your responsibilities and salary request as a community manager in Q2. However, there are mentions of the whole ambassador team's incentives, NFTs, and other things that seem like two different topics mixed in one proposal. 

I suggest that this proposal only focuses on your specific goals, plans, and funding requests as a community manager in Q2.

**Comment by hackfisher** [comment]
I would request one more thing to delivery, monthly report summaries key results.

And I'm afraid DCDAO may not going to manage Tron wallet, so better to change to some other network such as Arbitrum.

Better weekly update is always a good thing to have.

**Comment by furoxr** [comment]
https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xdb3c71b29674db9b1ce377ad22e5630bcfa8ffae5f17289dc2a9277b18a675bd
